test/test-integration/src/test/java/org/wso2am/micro/gw/tests/util/ApiProjectGenerator.java

```diff
@@ -85,13 +85,26 @@ public static String createApictlProjZip(String apiYamlPath, String swaggerPath,
             apiYamlFile);
         
         //swagger.yaml
-        Utils.copyFile(testResourcesPath + swaggerPath, apisZipPath + File.separator + definitions + File.separator +
-            openAPIFile);
+        if(swaggerPath != null){
+            Utils.copyFile(testResourcesPath + swaggerPath, apisZipPath + File.separator + definitions + File.separator +
+                    openAPIFile);
+        }
 
         if (certificatePath != null) {
             Utils.copyFile(testResourcesPath + certificatePath, 
                 apisZipPath + File.separator + endpointCertificates + File.separator + "backend.crt");
         }
+//        if(apiYamlPath != null){
```

Comment:
Add a comment mentioning the reason why L97-L107 has been commented

Reply:
Removed the commented block